ejb3/src/main/resources/schema/wildfly-ejb3_5_0.xsd

```diff
@@ -228,6 +228,7 @@
         <xs:attribute name="name" type="xs:string" use="required"/>
         <xs:attribute name="max-pool-size" type="xs:positiveInteger" default="20" use="optional"/>
         <xs:attribute name="derive-size" type="xs:string" use="optional"/>
+        <xs:attribute name="derived-size" type="xs:int" use="optional"/>
```

Comment:
The derived-size is a runtime attribute, it must not appear in the XSD to configure the resource.